<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellerBookInfoForm</class>
 <widget class="QWidget" name="SellerBookInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	font-size:20px;
	background-image: url(:/images/loginpushButton.png);
 	background-position: center ;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lb_author">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>101</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光细圆_CNKI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_type">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TE">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>461</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>隶书</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华光细黑_CNKI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>211</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string>图像</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_bookName">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>名字</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>185</y>
     <width>53</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>价格 ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_price">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>185</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_confirm">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	font-size:20px;
	background-image: url(:/images/loginpushButton.png);
 	background-position: center ;

}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	font-size:20px;
	background-image: url(:/images/loginpushButton.png);
 	background-position: center ;

}</string>
   </property>
   <property name="text">
    <string>选择图片</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_price">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_author">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_type">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
